plugins/sync.js

```diff
@@ -195,41 +195,97 @@ window.plugin.sync.RegisteredMap.prototype.loadDocument = function(callback) {
     setTimeout(function() {_this.loadDocument();}, window.plugin.sync.checkInterval);
   };
 
-  // this function called when the document is loaded
-  // update local data if the document is updated by other (overwrite!)
-  // and adding a timer to further check for updates
+  // this function gets called when the document is loaded
```

Comment:
'document' usually reference to `window.document` . Here we talk about a g-drive file